### DISCUSSION TITLE
Gostaria de ajuda pra conseguir colocar um status personalizado em meu bot

### DISCUSSION THREAD

**Discussion by HarizeTamara** [discordjs/discord.js#10643] [Q&A] [answered]
Gostaria de ajuda pra conseguir colocar um status personalizado em meu bot!
Tentei varias coisas mas não obtive o resultado que eu quero! 
![image](https://github.com/user-attachments/assets/c08faaec-faeb-4d33-ba56-09505d71a854)

**Comment by almostSouji** [comment, marked as answer]
Quoting my [FAQ](https://discord.com/channels/222078108977594368/1142187561729658942/1142187561729658942) entry from <https://discord.gg/djs>

> [!IMPORTANT]
> You need version **v14.13.0** or newer of discord.js
> Pick one option based on what other properties you want to set/change!

```js
const { ActivityType } = require('discord.js');
client.user.setPresence({
    activities: [{
         type: ActivityType.Custom,
         name: "custom", // name is exposed through the API but not shown in the client for ActivityType.Custom
         state: "🍋"
    }]
})
```
```js
const { ActivityType } = require('discord.js');
client.user.setActivity({
    type: ActivityType.Custom,
    name: "custom", // name is exposed through the API but not shown in the client for ActivityType.Custom
    state: "🍋"
})
```

```js
const { Client, GatewayIntentBits, ActivityType } = require('discord.js');
new Client({
    intents: [GatewayIntentBits.Guilds],
    presence: {
        activities: [{
            type: ActivityType.Custom,
            name: "custom", // name is exposed through the API but not shown in the client for ActivityType.Custom
            state: "🍋"
        }]
    }
});
```

- :information_source: Emojis are currently __not supported__ in custom status for bots (you can put an emoji as state, as shown above, but it will be part of the state, not a proper custom status emoji)
- :information_source: You can also supply `state` with the other activity types for an additional surface to display custom text in your bot's activity!

**Reply by HarizeTamara** [reply]
Thanks!!!
